### ISSUE TITLE
paremeter question

### ISSUE THREAD

**Issue by yasohasakii** [marl/autopool#4]
hi, I want to figure out if the ` instance_pred` in the code `bag_pred = AutoPool(axis=1, kernel_constraint=keras.constraints.max_norm(alpha_max, axis=0))(instance_pred)`  is embedded feature maps of multi-instances?  thanks! By the way it would more convenient if a demo script can be offer.

**Comment by bmcfee** [comment]
> is embedded feature maps of multi-instances?

They're not feature maps.  It's actually the instance-wise predictions (class likelihoods), ie the thing being pooled by autopool.



>  By the way it would more convenient if a demo script can be offer.

I completely agree!  I'll keep this issue open as a to-do.
